seriously-simple-stats.php

```diff
@@ -27,13 +27,13 @@
 
 use SeriouslySimpleStats\Classes\Stats;
 
-require_once 'vendor/autoload.php';
-
 define( 'SSP_STATS_VERSION', '1.3.0' );
 define( 'SSP_STATS_DIR_PATH', trailingslashit( plugin_dir_path( __FILE__ ) ) );
```

Comment:
Additionally... this expands to this:
> `define( 'SSP_STATS_DIR_PATH', trailingslashit( trailingslashit( dirname( __FILE__ ) ) ) );`

So it could just be:
```suggestion
define( 'SSP_STATS_DIR_PATH', __DIR__ . '/' );
```